src/main/java/pl/psnc/dei/ui/components/imports/CreateImportComponent.java

```diff
@@ -0,0 +1,256 @@
+			statusLayout.add(new Label("Import status"));
+			statusLayout.add(new Label(anImport.getStatus().toString()));
+			add(statusLayout);
+		}
+		refresh();
+	}
+
+	private Component generateSwitchingButtons() {
+		VerticalLayout switchingButtons = new VerticalLayout();
+		switchingButtons.setMaxWidth("100px");
+		Button addToSelected = new Button("->");
+		addToSelected.addClickListener(e -> {
+			List<Record> waitingForMovingToSelected = new ArrayList<>(allRecordsGrid.getSelectionModel().getSelectedItems());
+			selectedRecordsForImport.addAll(waitingForMovingToSelected);
+			allRecords.removeAll(waitingForMovingToSelected);
+			refresh();
+		});
+		switchingButtons.add(addToSelected);
+
+		Button moveFromSelectedToAll = new Button("<-");
+		moveFromSelectedToAll.addClickListener(e -> {
+			List<Record> waitingForMovingToAll = new ArrayList<>(selectedRecordsGrid.getSelectionModel().getSelectedItems());
+			allRecords.addAll(waitingForMovingToAll);
+			selectedRecordsForImport.removeAll(waitingForMovingToAll);
+			refresh();
+		});
+		switchingButtons.add(moveFromSelectedToAll);
+		return switchingButtons;
+	}
+
+	private Component generateActionButtons() {
+		HorizontalLayout actionButtons = new HorizontalLayout();
+
+		Button createButton = new Button("Create");
+		createButton.setEnabled(shouldShowCreateButton());
+		createButton.addClickListener(e -> {
+			if(selectedRecordsForImport.isEmpty()){
+				Notification.show("Import cannot be empty");
+				return;
+			}
+			importPackageService.createImport(importName.getValue(), project.getProjectId(), selectedRecordsForImport);
+		});
+		actionButtons.add(createButton);
+
+		Button updateButton = new Button("Update");
+		updateButton.setEnabled(shouldShowUpdateButton());
+		updateButton.addClickListener(e -> {
+			if(selectedRecordsForImport.isEmpty()){
+				Notification.show("Import cannot be empty");
+				return;
+			}
+			importPackageService.updateImport(anImport, selectedRecordsForImport);
+		});
+		actionButtons.add(updateButton);
+
+		Button sendButton = new Button("Send");
+		sendButton.setEnabled(shouldShowSendButton());
+		sendButton.addClickListener(e -> {
+			try {
+				importPackageService.sendExistingImport(anImport.getName());
+			} catch (NotFoundException ex) {
+				Notification.show("Something goes wrong");
+				ex.printStackTrace();
+			}
+		});
+		actionButtons.add(sendButton);
+
+		return actionButtons;
+	}
+
+	private boolean shouldShowSendButton() {
+		return anImport != null && ImportStatus.CREATED.equals(anImport.getStatus());
+	}
+
+	private boolean shouldShowUpdateButton() {
+		return anImport != null && !(ImportStatus.IN_PROGRESS.equals(anImport.getStatus()) && ImportStatus.SENT.equals(anImport.getStatus()));
+	}
+
+	private boolean shouldShowCreateButton() {
+		return anImport == null;
+	}
+
+	private Grid<Record> generateRecordsGrid(Set<Record> records) {
+		Grid<Record> recordsGrid = new Grid<>();
+		recordsGrid.setWidth("600px");
```

Comment:
Generally I would avoid putting sizes and other visual settings in code. It's better to move them to css and eventually change them later on that level.